verification/certifyKey_test.go

```diff
@@ -124,15 +423,210 @@ func testCertifyKey(d TestDPEInstance, t *testing.T) {
 	certifyKeyReq := CertifyKeyReq[SHA256Digest]{
 		ContextHandle: [16]byte{0},
 		Flags:         0,
```

Comment:
The if condition below covers for both test cases.

> 
> 	if flagIsCA == c.IsCA {
> 		t.Logf("ADD_IS_CA is set to %v and the basic constraint IsCA is set to %v", flagIsCA, c.IsCA)
> 	} else {
> 		// Fail when ADD_IS_CA flag is set properly in input parameters
> 		t.Errorf("ADD_IS_CA is set to %v but the basic constraint IsCA is set to %v", flagIsCA, c.IsCA)
> 	}

Reply:
Right, but it seems to me that we only actually run a test where ADD_IS_CA is false. Am I missing something?

Reply:
Ok, are you trying to convey that, the certifyKey request  should be built with different "flag" field (Flag="30th bit unset", Flag="30th bit set") and tested as two different functions "testCertifyKeyWithAddCA()", without "testCertifyKeyWithoutAddCA() instead of single testCertifyKey()"?

Reply:
Which way would be the best? - a test function for each input combination / single function for all input parameter combinations?

Reply:
Reg the set/unset ADD_IS_CA flag, I thought it could be implemented as part of third activity where the combination of input params are dealt.
To address your comment, I have created an array of inputs (CertifyKeyRequests), as we usually do in Go tests - iterated  validated.
Since, setting ADD_IS_CA=True in CertifyKeyReq.Flags does not make isCA=True in certificate, so I have logged instead of throwing error.
I have added a TODO comment mentioning this issue.